README.md

```diff
@@ -1,2 +1,2 @@
-# OOP-School-library
-This project is about implementing classes to represent students and teachers in the Library
+# OOP-School-library

```

Comment:
- [ ] Good job with the README, however, you are still missing some sections in the template. Kindly add the missing sections according to the [Microverse template](https://github.com/microverseinc/readme-template). All sections are required but you can skip the Deployment, Tests, and Demo link parts if you don't have them yet.